apps/src/sites/studio/pages/googleBlocklyWrapper.js

```diff
@@ -0,0 +1,191 @@
+const BlocklyWrapper = function(blocklyInstance) {
+  this.blockly_ = blocklyInstance;
+  this.Msg = blocklyInstance.Msg;
+  this.inject = this.blockly_.inject;
+  this.wrapReadOnlyProperty = function(propertyName) {
+    Object.defineProperty(this, propertyName, {
+      get: function() {
+        return this.blockly_[propertyName];
+      }
+    });
+  };
+  this.wrapSettableProperty = function(propertyName) {
+    Object.defineProperty(this, propertyName, {
+      get: function() {
+        return this.blockly_[propertyName];
+      },
+      set: function(newValue) {
+        this.blockly_[propertyName] = newValue;
+      }
+    });
+  };
+};
+
+function initializeBlocklyWrapper(blocklyInstance) {
+  const blocklyWrapper = new BlocklyWrapper(blocklyInstance);
+
+  blocklyWrapper.wrapReadOnlyProperty('ALIGN_CENTRE');
```

Comment:
Worth documenting where this list came from?

Reply:
I think it's probably not worth adding to the code itself, but documenting here:
this list was compiled by grepping for `Blockly.` and `blockly.` in all of our apps code, and just seeing what Blockly properties we use. The results are captured in this spreadsheet, which also determines which properties need to be settable and which can be read-only: https://docs.google.com/spreadsheets/d/17c0VRX9GV9SKTxbfKPoPmlSLWT5UoWzsGroSts3wuzE/edit#gid=676426278